examples/controlnet-interior-design/components/generate_prompts/fondant_component.yaml

```diff
@@ -0,0 +1,14 @@
+name: Generate prompts
```

Comment:
Also a kubeflow component spec is missing for now, but I assume the pipeline wrapper of #34 is going to add that

Reply:
Yes, we shouldn't have to add one to the code, it should be handled by Fondant.